src/openedx_companion_auth/README.rst

```diff
@@ -0,0 +1,134 @@
+Open edX auth companion
+=======================
+
+An Openedx auth companion plugin that redirects users from edx-platform to MIT applications
+
+Prerequisites
+-------------
+
+- Run the following commands to install pants locally:
+
+  .. code-block:: shell
+
+     curl --proto '=https' --tlsv1.2 -fsSL https://static.pantsbuild.org/setup/get-pants.sh | bash
+
+     export PATH=$PATH:/root/.local/bin
+
+Building
+--------
+
+- Build a pip-installable package into `dist/`:
+
+  .. code-block:: shell
+
+     pants package ::
+
+Installing
+----------
+
+All that is required to install this in either a hosted version of Open edX or devstack is a `pip install` either from PyPI or the output of `pants build`.
+
+Run Tests
+---------
+
+- Clone the open-edx-plugins repository in the src folder present in the openedx work directory.
+- Build the packages using pants:
+
+  .. code-block:: shell
+
+     pants package ::
+
+- In the devstack folder, run:
+
+  .. code-block:: shell
+
+     make lms-shell
+
+- In the lms shell, activate the edxapp virtual environment:
+
+  .. code-block:: shell
+
+     source /edx/app/edxapp/venvs/edxapp/bin/activate
+
+- Install the package generated by pants in the dist folder using pip:
+
+  .. code-block:: shell
+
+     pip install /edx/src/open-edx-plugins/dist/openedx-companion-auth-{version}.tar.gz
+
+- Go to the package directory:
+
+  .. code-block:: shell
+
+     cd open-edx-plugins/src/openedx_companion_auth
```

Comment:
I think all these steps would moved under the Installation heading, You can check the other plugin READMEs for reference. e.g. https://github.com/mitodl/open-edx-plugins/blob/8142a0c654150c21de2602024cc9bb5c2d12fc49/src/edx_sysadmin/README.rst